plugins/woocommerce-beta-tester/woocommerce-beta-tester.php

```diff
@@ -88,6 +88,8 @@ function add_extension_register_script() {
 		);
 	$script_url        = plugins_url( $script_path, __FILE__ );
 
+	$script_asset['dependencies'][] = WC_ADMIN_APP; // Add WCA as a dependency to ensure it loads first.
+
```

Comment:
I did this to ensure `window.wcAdminFeatures` is created before the script runs.